docs/developer-docs/setup/pocket-ic.md

```diff
@@ -0,0 +1,278 @@
+# Testing canisters with PocketIC
+
+## Overview
+
+PocketIC is a lightweight, deterministic testing platform for programmatic testing for canisters written in Rust and Python. It seamlessly integrates with existing testing infrastructure, such as `cargo test`, and runs as a standalone binary that doesn't require additional containers or virtual machines. Currently, PocketIC is supported on MacOS and Linux host systems. 
```

Comment:
Just a suggestion, but I think "for programmatic testing for canisters written in Rust and Python" would sound a bit better as "for programmatically testing canisters written in Rust and Python".

On an unrelated note, there is also TypeScript support via [PicJS](https://github.com/hadronous/pic-js). I'd be happy to contribute an additional section to cover that.